meta.yml

```
@@ -3,16 +3,19 @@ fullname: Stalmarck
 shortname: stalmarck
 organization: coq-community
 community: true
+travis: true
 
 synopsis: Proof of Stålmarck's algorithm in Coq
 
 description: |
   A two-level approach to prove tautologies using Stålmarck's algorithm in Coq.
 
+doi: 10.1007/3-540-44659-1_24
+
 paper:
```

```suggestion
publications:
```
For the paper to show up in the README this needs to be changed (we initially assumed just one paper).